`mint` command + minor updates for `collect-reward` in CLI (#2097)
nucypher/nucypher: docs/source/guides/network_node/staking_guide.rst

Note that you will need to confirm two transactions if you collect both types of
staking compensation if you use a hardware wallet.

.. note:: If you want to withdraw all tokens when all of them are unlocked - 

>           make sure to call ``nucypher stake mint`` before that to obtain last reward too

Reviewer: ```suggestion
          make sure to call ``nucypher stake mint`` first to ensure the last reward is included
```
